portalnet/src/overlay_service.rs

```diff
@@ -1241,6 +1242,23 @@ where
         let utp = Arc::clone(&self.utp_socket);
         let metrics = self.metrics.clone();
 
+        let mut content_keys_string = vec![];
+        if enabled!(Level::ERROR) {
```

Comment:
Are there really any situations when any user would not be running `ERROR` level logs?

Reply:
I wasn't 100 percent sure if there isn't i will remove the if statements. I seen errors were enabled by default though

Reply:
I think it's safe to assume users will always be running at the bare minimum `ERROR` level logs.

Reply:
Ok I will update the PE in 10 minutes with that change